gslb/ingestion/fullsync.go

```diff
@@ -399,14 +399,16 @@ func GenerateModels(gsCache *avicache.AviCache) {
 			continue
 		}
 		tenant, hmName := hmKey.Tenant, hmKey.Name
-		gsName, err := avicache.GetGSFromHmName(hmName)
+		gsName, gen, err := avicache.GetGSFromHmName(hmName)
 		if err != nil {
-			gslbutils.Debugf("key: %v, msg: can't get gs name from hm, err : %v", hmKey, err)
-			continue
+			gslbutils.Logf("key: %v, msg: can't get gs name from hm, err : %v", hmKey, err)
```

Comment:
It looks like there is a possibility that the `GetGSFromHmName` method returns an error but we won't hit the `continue` block in Line number 406. This seems to be different from the old commit.

Reply:
We do hit line 406 for health monitors for passthrough HMs and HMs not created by amko ( System HMs like - System-GSLB-HTTP, System-GSLB-TCP, System-GSLB-UDP, etc)

Reply:
Right, the `if` condition was not necessary, removed it